modules/visual_script/visual_script_editor.cpp

```diff
@@ -2153,7 +2153,7 @@ void VisualScriptEditor::goto_line(int p_line, bool p_with_error) {
 			_update_graph();
 			_update_members();
 
-			call_deferred("_center_on_node", p_line); //editor might be just created and size might not exist yet
+			call_deferred("call_deferred", "_center_on_node", p_line); //editor might be just created and size might not exist yet
```

Comment:
`VisualScriptEditor::_center_on_node` -> `graph->set_scroll_ofs` -> `GraphEdit::_update_scroll` -> `call_deferred("_update_scroll_offset");`.
This happens at least the first time before you open the function in the script editor.
Yes, this is not a good fix, next it will be `call_deferred("call_deferred", "call_deferred", "call_deferred", "whatever")`.
I hope someone knows how to actually fix it.